spec/spec_helper.rb

```diff
@@ -1,18 +1,56 @@
-# This file is managed via modulesync
```

Comment:
Please do a rebase against the master branch, because this file is managed by ```voxpupuli/modulesync```, ```voxpupuli/modulesync_config```.